templates/guides/migration-guide-1.0.0.md

```diff
@@ -0,0 +1,79 @@
+---
+page_title: "Fastly Terraform 1.0.0 Migration Guide"
+subcategory: "Migration"
+---
+
+## Fastly Terraform 1.0.0 Migration Guide
+
+**Resources omit `_v1` suffix**:
+
+All resources now have a consistent naming convention which omit the previous v1 suffix.
+
+- `fastly_service_v1` -> `fastly_service_vcl` (renamed with `_vcl` suffix)
```

Comment:
"renamed with vcl suffix" says what happened but that's fairly evident. It may be more useful to say why, eg "(Fastly has two service types; this resource has always described the VCL one)